apps/vth-frontend/src/app/[locale]/layout.tsx

```diff
@@ -73,9 +73,9 @@ const RootLayout = async ({ children, params: { locale } }: LayoutProps) => {
 
   const navigationPages: PageData[] = data?.navigationPages?.data;
 
-  navigationPages?.sort((a, b) => {
-    return a.attributes.title.localeCompare(b.attributes.title);
-  });
+  // navigationPages?.sort((a, b) => {
+  //   return a.attributes.title.localeCompare(b.attributes.title);
+  // });
```

Comment:
@AliKdhim87 @savitris Ik denk dat het nog steeds handig is om in deze JavaScript te sorteren op de `order` propertie, na eerst te sorteren met `localeCompare`. Op die manier blijft de volgorde van navigationPages zonder `order` nog steeds stabiel, en hoeven we niet `order` verplicht te maken.

Reply:
How about we achieve that by adding this `(sort: ["order:asc", "title:asc"] )` to the query?
https://docs.strapi.io/dev-docs/api/graphql#sorting